tools/ci/travis/travis-ci.sh

```diff
@@ -157,6 +157,57 @@ EOF
     testsuite > /dev/null && exit 1 || echo pass
 }
 
+CheckTypo () {
+  export OCAML_CT_HEAD=$1
+  export OCAML_CT_LS_FILES="git diff-tree --no-commit-id --name-only -r $2 --"
+  export OCAML_CT_CAT="git cat-file --textconv"
+  export OCAML_CT_PREFIX="$1:"
+  GIT_INDEX_FILE=tmp-index git read-tree --reset -i $1
+  git diff-tree --no-commit-id --name-only -r $1 | (while IFS= read -r path
+  do
+    echo "Checking $1: $path"
+    if ! tools/check-typo $path ; then
+      touch check-typo-failed
+    fi
+  done)
+  rm -f tmp-index
+}
+
+CHECK_ALL_COMMITS=0
+
+CheckTypoAllCommits () {
```

Comment:
Could you find a better name for this function? It doesn't necessarily check all commits anymore.